pkg/pods/pod.go

```diff
@@ -67,7 +70,6 @@ func NewPod(id types.PodID, path string) *Pod {
 		ServiceBuilder: runit.DefaultBuilder,
 		P2Exec:         DefaultP2Exec,
 		DefaultTimeout: 60 * time.Second,
-		LogExec:        defaultLogExec,
```

Comment:
I think we'll always want to provide defaults here. Not all `Pod`s are constructed by the preparer / in `orchestrate.go`. As it stands, you can't just build a default `Pod` and safely launch it, whereas previously you could.